packages/components/breadcrumb/src/breadcrumb.tsx

```diff
@@ -139,35 +144,46 @@ export const Breadcrumb = forwardRef<BreadcrumbProps, "nav">((props, ref) => {
       return validChildren.map((child, index) =>
         cloneElement(child, {
           gap,
-          isLastChild: validChildren.length === index + 1,
+          lastChild: validChildren.length === index + 1,
           separator,
         }),
       )
     } else {
       let hiddenEllipsis: BreadcrumbGenerateItem[] = []
 
       return items.map((item, index) => {
-        const { name, isCurrentPage, isEllipsisPage, containerProps, ...rest } =
-          item
-        const isLastChild = items.length === index + 1
+        let {
```

Comment:
@nathanlao 

I was wondering why you are using `let` to destructure everything except `currentPage` and `ellipsisPage`.

Here is an example:

```diff
- let {
-   name,
-   currentPage,
-   ellipsisPage,
-   isCurrentPage,
-   isEllipsisPage,
-   containerProps,
-   ...rest
- } = item
- const lastChild = items.length === index + 1
- 
- currentPage ??= isCurrentPage
- ellipsisPage ??= isEllipsisPage
+ const {
+   name,
+   currentPage: _currentPage,
+   ellipsisPage: _ellipsisPage,
+   isCurrentPage,
+   isEllipsisPage,
+   containerProps,
+   ...rest
+ } = item
+ const lastChild = items.length === index + 1
+ 
+ let currentPage = _currentPage
+ let ellipsisPage = _ellipsisPage
+ 
+ currentPage ??= isCurrentPage
+ ellipsisPage ??= isEllipsisPage
```

Reply:
Ohh! Thank you for your sugguestion! I will use this method to avoid using `let` for my later PR!

Plz re-review :)

Reply:
@koralle @nathanlao 

```
let currentPage = _currentPage
let ellipsisPage = _ellipsisPage
```

There is no need to do so. I would like to see it put back together with the others.

Reply:
@koralle @hirotomoyamada 
Changes have been reverted, please re-review :)